sdk/python/examples/bayesianoptimization-katib-sdk.ipynb

```diff
@@ -0,0 +1,575 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/root/katib/sdk/python\n"
+     ]
+    }
+   ],
+   "source": [
+    "cd /root/katib/sdk/python"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! python setup.py install"
```

Comment:
I am not sure that we need this setup script in Example, since we already mentioned how to install SDK in README.

Reply:
removed from examples